### DISCUSSION TITLE
generated columns

### DISCUSSION THREAD

**Discussion by bluebrown** [aarondl/sqlboiler#1047] [Q&A] [answered]
Hi, I noticed that sqlboiler does  not include generated columns in the struct for generated columns. With the below schema, end_date is missing from all boilerplate code. Is there a some setting or workaround for this scenario?

```sql
CREATE TABLE ITEM (
  ID  SERIAL PRIMARY KEY,
  RELEASE_DATE date,
  END_DATE date GENERATED ALWAYS AS (
    CASE
      WHEN RELEASE_DATE IS NOT NULL THEN RELEASE_DATE + interval '60 month'
      ELSE NULL
    END
  ) STORED
);

**Comment by bluebrown** [comment]
Is there any chance to make this work? I have not found an answer yet.  On stackoverflow I also did not get an answer  https://stackoverflow.com/questions/70404037/generated-columns-with-sqlboiler.

**Comment by stephenafamo** [comment, marked as answer]
Generated columns are currently ignored by SQLBoiler because they were generating problems in earlier versions.

Now it is feasible to re-enable them, so we will work on this.

**Reply by stephenafamo** [reply]
This will be in the next release 4.8.4

**Reply by bluebrown** [reply]
Amazing, thank you.
